ext/standard/streamsfuncs.c

```diff
@@ -1627,7 +1627,7 @@ PHP_FUNCTION(stream_isatty)
 {
 	zval *zsrc;
 	php_stream *stream;
-	zend_long fileno;
+	int fileno;
```

Comment:
As `php_socket_t` is used to perform the write, shouldn't we be using it here as well? On Windows this is typedef'd to `SOCKET` (https://github.com/php/php-src/blob/a7fe2570d3ce6915d4ea85c62c0f880ddc225ba7/main/php_network.h#L99) which AFAIK is a 64-bit type, so using int here could cause stack corruption.